yb-voyager/src/srcdb/oracle.go

```diff
@@ -560,6 +560,11 @@ func (ora *Oracle) GetColumnsWithSupportedTypes(tableList []sqlname.NameTuple, u
 			//Using this ContainsAnyStringFromSlice as the catalog we use for fetching datatypes uses the data_type only
 			// which just contains the base type for example VARCHARs it won't include any length, precision or scale information
 			//of these types there are other columns available for these information so we just do string match of types with our list
+			// Experimental flag: allow CLOB export in offline (ora2pg) path only
+			if bool(ora.source.AllowOracleClobDataExport) && !useDebezium && !isStreamingEnabled && strings.EqualFold(dataTypes[i], "CLOB") {
+				supportedColumnNames = append(supportedColumnNames, columns[i])
+				continue
+			}
```

Comment:
Alternatively, you can modify the `OracleSupportedDataTypes` list in the starting of the function to remove CLOB from it if this flag is enabled, you don't need `useDebezium` check as well, as we know this flag will ensure that this flag is not allowed for that